js/notify/electron-notify.js

```diff
@@ -208,12 +227,21 @@ function calcDimensions() {
 function setupConfig() {
     closeAll();
 
-    // Use primary display only
-    let display = electron.screen.getPrimaryDisplay();
+    // This feature only applies to windows
+    if (!isMac) {
```

Comment:
@lneir Fixed ```first``` & ```third``` points. For the ```second``` point this feature is restricted to work only on windows.